pyproject.toml

```diff
@@ -7,11 +7,8 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-fastapi = "^0.57.0"
```

Comment:
Removing these 3 lines is causing the project build to break.
please revert this change

Reply:
@kamaldodiya Did you try to build a new docker image?
20.82 Warning: poetry.lock is not consistent with pyproject.toml. You may be getting improper dependencies. Run `poetry lock [--no-update]` to fix it.
34.28 
34.28 Because ensembl-web-metadata-api depends on starlette (^0.13.4) which doesn't match any versions, version solving failed.
------
failed to solve: process "/bin/sh -c pip install poetry &&     poetry config virtualenvs.create false &&     poetry install --no-dev" did not complete successfully: exit code: 1